src/api/ha_civo/main.go

```diff
@@ -77,25 +77,40 @@ func fetchAPIKey() string {
 // TODO demoScript try the script whether its working or not
 func demoScript() string {
 	return `#!/bin/bash
-sudo apt update -y
-sudo apt install nginx git -y
+sudo apt update -y 
+sudo apt install -y nginx git
 `
 }
```

Comment:
Using this function to get the PostInstall script to install stuff
Problem is its not working 😢 
does anyone know what to do